airbyte-integrations/connectors/source-asana/source_asana/streams.py

```diff
@@ -81,7 +89,7 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
         response_json = response.json()
         yield from response_json.get("data", [])  # Asana puts records in a container array "data"
 
-    def read_slices_from_records(self, stream_class: Type[AsanaStream], slice_field: str) -> Iterable[Optional[Mapping[str, Any]]]:
+    def read_slices_from_records(self, stream_class: object, slice_field: str) -> Iterable[Optional[Mapping[str, Any]]]:
```

Comment:
why object?

Reply:
What type-hint would you like?

Reply:
the one we used to have was good enough
https://docs.python.org/3/library/typing.html#typing.Type

Reply:
This one was not accessible, removed it from type hints, it's just useless and not pointing to anything. But if you say "revert it" - no problem.

Reply:
well it's useless at runtime as well as all the type hinting :) 
but since we do use type hints, I believe this one suited the best here, so yes, lets revert it if you dont mind